engines/router/missionctl/log/resultlog/upi_result_log.go

```diff
@@ -142,49 +250,11 @@ func (ul *UPIResultLogger) LogTuringRouterRequestSummary(
 
 	// Log the responses. If an error occurs in logging the result to the
 	// configured result log destination, log the error.
-	if err := ul.LogEntry(routerLog); err != nil {
+	if err := ul.logEntry(routerLog); err != nil {
 		log.Glob().Errorf("Result Logging Error: %s", err.Error())
```

Comment:
should we log the `routerLog` so we know which message that needs to manually retry?